src/Vite.AspNetCore/TagHelpers/ViteTagHelper.cs

```diff
@@ -112,10 +112,25 @@ public override void Process(TagHelperContext context, TagHelperOutput output)
 			// If the Vite script was not inserted, it will be prepended to the current element tag.
 			if (ViteStatusService.IsMiddlewareRegistered && !this._status.IsDevScriptInserted)
 			{
+				var baseUrl = CombineUri(this._viteOptions.GetViteDevServerUrl(), this._viteOptions.Base ?? string.Empty);
 				// Build the uri to the vite client script.
-				var viteClientUrl = this._viteOptions.GetViteDevServerUrl() + "/@vite/client";
+				var viteClientUrl = baseUrl + "@vite/client";
 				// Add the script tag to the output
 				output.PreElement.AppendHtml($"<script type=\"module\" src=\"{viteClientUrl}\"></script>");
+
+				if (this._viteOptions.UseReactRefresh)
+				{
+					var viteReactRefreshUrl = baseUrl + "@react-refresh";
+					output.PreElement.AppendHtml(
+							"<script type=\"module\">\n" +
+							"	import RefreshRuntime from \"" + viteReactRefreshUrl + "\";\n" +
```

Comment:
This is like before. If a base path like `/dist/` is used. The final url will be something like `http://localhost:5173/dist/@react-refresh` instead of `http://localhost:5173/@react-refresh`. Is that correct?